mapit_gb/management/commands/mapit_UK_add_2020_structural_changes.py

```diff
@@ -0,0 +1,72 @@
+# coding=utf8
+
+from django.core.management.base import BaseCommand
+from mapit.models import Area, Generation, Type, Country, NameType, CodeType
+
+
+class Command(BaseCommand):
+    help = "Sort out boundaries for 2020 council changes"
+
+    def add_arguments(self, parser):
+        parser.add_argument('--commit', action='store_true')
+
+    def handle(self, *args, **options):
+        self.commit = options['commit']
+        self.g = Generation.objects.current()
+        self.gn = Generation.objects.new()
+        self.name_type = NameType.objects.get(code='O')
+        self.code_type = CodeType.objects.get(code='gss')
+        self.country = Country.objects.get(code='E')
+
+        self.raise_generation_on_everything_else()
+
+        self.areas = Area.objects.filter(generation_low_id__lte=self.g, generation_high_id__gte=self.g)
+
+        self.get_existing()
+        self.buckinghamshire()
```

Comment:
I think this line will crash because `self.buckinghamshire` isn't a method any more - `get_existing` has set that instance variable to an `Area`. I guess renaming the method to `create_buckinghamshire` or similar is required.